mgnify_pipelines_toolkit/analysis/assembly/process_dbcan_result_cazys.py

```diff
@@ -84,14 +84,13 @@ def print_gff(overview_file, outfile, dbcan_version, substrates, genome_gff_line
         with fileinput.hook_compressed(overview_file, "r", encoding="utf-8") as file_in:
             for line in file_in:
                 if line.startswith("MGYG") or line.startswith("ERZ"):
```

Comment:
You could invert this line to skip it instead of nesting the whole thing:
```python
if not line.startswith("MGYG") and not line.startswith("ERZ"):
   continue
```